otterdog/eclipse-equinox.jsonnet

```
@@ -56,6 +57,25 @@ orgs.newOrg('eclipse.equinox', 'eclipse-equinox') {
           value: "pass:bots/eclipse.equinox/github.com/token-hd5020",
         },
       ],
+      rulesets: [
+        orgs.newRepoRuleset('master') {
+          allows_updates: true,
+          bypass_actors+: [
+            "@eclipse-equinox/eclipseequinox-project-committers"
+          ],
+          include_refs+: [
+            "refs/heads/master",
+          ],
+          required_approving_review_count: 0,
+          required_status_checks+: [
+            "continuous-integration/jenkins/pr-head",
+            "call-license-check / check-licenses"
+          ],
```

The validation error you can see in https://github.com/eclipse-equinox/.eclipsefdn/pull/7#issuecomment-3407430743 comes from here. The `required_status_checks` is an object, not an array. It should be something like:

```suggestion
          required_status_checks+: {
              strict: true,
              status_checks+: [
                "continuous-integration/jenkins/pr-head",
                "call-license-check / check-licenses"
              ]
          },
```

See https://otterdog.readthedocs.io/en/latest/reference/organization/repository/ruleset/ for details